.eslintrc

```diff
@@ -1,19 +1,35 @@
 {
   "extends": [
-    "react-app",
-    "prettier/@typescript-eslint",
-    "plugin:prettier/recommended"
-  ],
-  "plugins": [
-    "simple-import-sort"
+    "prettier",
+    "eslint:recommended",
+    "plugin:prettier/recommended",
+    "plugin:import/typescript",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended"
   ],
+  "plugins": ["simple-import-sort", "@typescript-eslint", "prettier", "import"],
   "rules": {
     "simple-import-sort/imports": "error",
     "simple-import-sort/exports": "error"
   },
   "settings": {
-    "react": {
-      "version": "999.999.999"
+    "import/parsers": {
+      "@typescript-eslint/parser": [".ts"]
+    },
+    "import/resolver": {
+      "node": {
+        "extensions": [".js", ".ts"]
+      }
+    }
+  },
+  "overrides": [
+    {
+      "files": ["*.ts"],
+      "rules": {
+        "no-dupe-class-members": "off",
+        "@typescript-eslint/camelcase": "off",
```

Comment:
We should use camelcase in TypeScript, but saw in many places we were using snake case, disabled this but would be nice to just map snake case to camelCase moving forward. Thoughts?

Reply:
we use it a lot for decoding json responses, how do you think we should handle that?

Reply:
I have used serializers before, that are in charge of converting an external data structure to an internal domain interface, and through DTOs the other way around. It sometimes gets tedious though.

Reply:
Exactly what Gambito said, at work we use this package https://github.com/typestack/class-transformer but I guess we could try and implement our own if we don't want to add dependencies.

Reply:
Yeah happy to add that

Reply:
Awesome! We can create an issue for it, so we don't make this PR too big :)